editor/src/messages/portfolio/document/overlays/utility_types.rs

```diff
@@ -21,6 +21,108 @@ pub fn empty_provider() -> OverlayProvider {
 	|_| Message::NoOp
 }
 
+// Types of overlays used by DocumentMessage to enable/disable select group of overlays in the frontend
+#[derive(PartialEq, Clone, Debug, serde::Serialize, serde::Deserialize, specta::Type)]
+pub enum OverlaysType {
+	All,
```

Comment:
You can probably remove `All` and replace the whole `OverlaysType` with `Option<OverlaysType>` where `None` is equivalent to `All` being false and `Some` is equivalent to true.

Reply:
Upon further thought, that's not true because you'll lose your settings when it becomes `None`. I recommend just storing a separate bool outside of `OverlaysType`.